grouper/fe/handlers/service_account_view.py

```diff
@@ -0,0 +1,90 @@
+from grouper import group as group_biz
+from grouper.constants import USER_METADATA_SHELL_KEY
+from grouper.fe.handlers.user_disable import UserDisable
+from grouper.fe.handlers.user_enable import UserEnable
+from grouper.fe.util import GrouperHandler
+from grouper.graph import NoSuchUser
+from grouper.model_soup import (User, Group, APPROVER_ROLE_INDICIES, AUDIT_STATUS_CHOICES,
+        OWNER_ROLE_INDICES)
+from grouper.permissions import get_requests_by_owner
+from grouper.public_key import get_public_keys_of_user
+from grouper.user import user_open_audits, user_requests_aggregate, user_grantable_permissions, user_role, user_role_index
+from grouper.user_metadata import get_user_metadata_by_key
+
+
+class ServiceAccountView(GrouperHandler):
+    def get(self, user_id=None, name=None):
+        print("HELOO")
+        self.handle_refresh()
+        user = User.get(self.session, user_id, name)
+        if user_id is not None:
+            user = self.session.query(User).filter_by(id=user_id).scalar()
+        else:
+            user = self.session.query(User).filter_by(username=name).scalar()
+
+        if not user:
+            return self.notfound()
+
+        if not user.role_user:
+            return self.notfound()
+
+        can_control = user.name == self.current_user.name or self.current_user.user_admin
+        can_disable = UserDisable.check_access(self.session, self.current_user, user)
+        can_enable = UserEnable.check_access(self.session, self.current_user, user)
+
+        if user.id == self.current_user.id:
```

Comment:
fwiw this should always be false, since a service account shouldn't ever hit `grouper-fe`.